gravitee-apim-console-webui/src/management/api/proxy/health-check/api-proxy-health-check.component.ts

```diff
@@ -75,7 +75,12 @@ export class ApiProxyHealthCheckComponent implements OnInit, OnDestroy {
         onlyApiV2Filter(this.snackBarService),
         switchMap((api) => {
           const apiHealthCheck = ApiProxyHealthCheckFormComponent.HealthCheckFromFormGroup(this.healthCheckForm, false);
-          this.updateEndpointsHealthCheckConfig(api.proxy?.groups);
+          try {
```

Comment:
Is it really necessary if you fixed the issue on line 110 ? And shouln't we use rxjs catchError instead of a try catch if we would really want to catch a specific error ? 🤔 
I would have remove it tbh.

Reply:
Removed